.changelog/v0.20.0/summary.md

```diff
@@ -0,0 +1,19 @@
+*Jun 22, 2021*
+
+This release's number is bumped up to v0.20.0 due to two minor breaking changes
+in our public APIs (see the breaking changes section below for details).
+
+Also, since nobody was really making use of the Light Node, we decided to remove
+its crate from the repo for now. If anyone needs it back, please contact us and
+we'll restore it (although, we are considering migrating any and all binaries to
+their own repositories in future to separate framework-level concerns from
+operational ones).
+
+The `tendermint-p2p` crate is still undergoing significant expansion (thanks to
+@xla and @melekes!). A lot's been done and we're in the process of finalizing
```

Comment:
```suggestion
@xla and @melekes)! A lot's been done and we're in the process of finalizing
```

Reply:
The "thanks" part is parenthesized and emphasized :slightly_smiling_face:

Reply:
I was suggesting the change, because it breaks the link to the user profile.

Reply:
Ah, I see. Let's just take the exclamation mark out?

```suggestion
@xla and @melekes). A lot's been done and we're in the process of finalizing
```

Reply:
Sounds good.